lib/active_record/type/spatial.rb

```diff
@@ -40,7 +40,7 @@ def self.parse_sql_type(sql_type)
       end
 
       def spatial_factory
-        @spatial_factory ||=
```

Comment:
This was causing a test to fail. (See [here](https://app.travis-ci.com/github/SenteraLLC/activerecord-mysql2rgeo-adapter/jobs/568851516)

When saving the model, it was using the already initialized and incorrect spatial_factory. This was causing it the data to be persisted as the wrong type (RGeo::Geos::CAPIPointImpl instead of RGeo::Geographic::SphericalPointImpl).

Reply:
Do you have any performance concerns about no longer caching the factory? Is there a different way to clear the cached factory when you know it should be cleared, rather than reinitializing the `@spatial_factory` instance variable each time this method is called?

Reply:
I think that looks ok, thanks.